### ISSUE TITLE
Origin Questlines: Extinct Abductors

### ISSUE THREAD

**Issue by tzerrell** [MoreEventsMod/More_Events_Mod_Beta#109]
_Idea by Malthus_
You find a world with ruins of an ancient civilization but what buffles your scientists is the fact that they find remains of your own species among alien ones, later finding out that these were abducted and experimented on. These aliens resembling demons of your peoples myths

**Comment by tzerrell** [comment]
If desired, this could be connected to the event chain #72 that I'm working on implementing. Or it could stand on its own, either way seems reasonable (and there's no need to wait for me to make progress on #72 before starting this).

**Comment by Tuttu** [comment]
Indeed, both might be joined or kept separated. I don't know what could be best. 
Maybe to have the chance to get one of the others during your playthrough, like it works with the Precursors event chain?

**Comment by Elimdur** [comment]
I am going to further develop this idea.

My thought is to have ONE anomaly start with the player finding an ancient abandoned research facility on a remote planet.

You get a research project to get more informations what has happened there.

From there on we can have multiple branching storylines. 

1 . Possibility : Those aliens were evil and abducted different species to experiment on them to form slave races to serve their needs.

so this event continues to tell the player that this aliens bear striking similarities to demons of the player´s species' legends who were said to take people away never to return.

You´ll find coordinates of their homesystem in their computer systems. (Spawns one of multiple system initialisers)

Initialiser 1: A system with nuked planets and many debri fields
 in this case you can just assume they got what they deserved, maybe find something useful and move on.
Initialiser 2: A system with one planet and a pre spacefaring species (the same species that abducted the other species) surrounded by automated killer satellites which ensure this species never reaches for the stars again
 in this case it may get more interesting. You get an automated message telling you what this species did and why the creators of this killer satellites chose to build them instead of just destroying them completely. The creators have suffered from the things this species did to them but they were fanatic peaceful and confining this evil species to its own planet was the worst possible thing they could reconcile with their conscience. All this must have happened ages ago. The evil species civilisation has collapsed and multiple trys to enter space again have been denied by the satellite system which then decimated the population again sending them back into stoneage or something like this again. Todays population knows nothing about the crimes of their ancestors.
The player then gets to choose what to do. Either reprogramming the sattelites to kill everything on the planet, to letting everything as it is and go, to destroy the sattelites and let them get a second chance maybe even be their benefactor and enlighten them.
Initialiser X: Maybe you come up with more possibilities

2 . Possibility: They were benevolent beings that uplifted yours and several other species and aided in there development.

You may then also get coordinates of their homesystem with different initialisers

One could be a system with a nuked planet as well.

Another possibility could be a system with a shielded planet housing all their technological achievements stored there until one of the young species may be worthy to receive them. The species itself might have ascended (like the ancients in stargate) having only left a message for those coming after them.

Maybe you´ll have some more ideas for other branches.

**Comment by Talanic** [comment]
I'm on this.  One chain is written already.  See https://docs.google.com/document/d/17RPHagGg4YlZx9SXpps0BP-O8L95XviWrKcB3-kx-2A/edit?usp=sharing for the work in progress.

**Comment by Elimdur** [comment]
Just a temporary save location:

Who are you?

I am Chap'Lin, the last of the Orila. Though I am merely an image of his consciousness and not the person itself. He was once chosen by his people to represent our species to those who would follow in our footsteps.

What are you?

I am an image of the consciousness of a chosen Orila. I am more than a simple AI but less than a living being. To be what I am is a honor as well as a burden...A burden I readily accepted. Since the last of our race passed away it has been my responsibility to maintain what has been left of this world and to ensure the continuous functionality of its planetary barrier shield. But my most important task was to wait for your arrival.

Can you describe the society of your empire?

We did not want to have an empire. That was not our way. We did not want to rule or conquer anyone. We would have liked to help everyone if possible. All intelligent life should want to help one another. We all want to live by each others' happiness, not by each other's misery. We don't want to hate and despise one another. We believed that in this universe there was room for everyone. And that our galaxy was rich and could provide for everyone. That the way of life could have been free and beautiful. But some had lost the way...

Lost the way?

Greed had poisoned their souls; had barricaded their world with hate; had goose-stepped them into misery and bloodshed. They had developed speed, but had shut themselves in. Machinery that gives abundance had left them in want. Their knowledge had made them cynical; their cleverness, hard and unkind. They thought too much and felt too little. More than machinery would have needed compassion. More than cleverness, they would have needed kindness and gentleness. Without these qualities, their way of life was violent and they threatened the life of others.
